app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,53 @@
+class CommentsController < ApplicationController
+  include ApplicationHelper
+
+  # def new
+  #   @comment = Post.new(user_id: current_user.id, movie_id: params[:user_id])
+  # end
+
+  def create
+    @comment = Comment.new(comment_params)
```

Comment:
Be sure to get the new comment's user from `current_user`, not from the form (since the form data can be spoofed with a command line tool like cURL).
```ruby
@comment = Comment.new(comment_params)
@comment.user = current_user
```